.github/workflows/azure-static-web-apps-calm-smoke-0bd84271e.yml

```diff
@@ -0,0 +1,56 @@
+name: Azure Static Web Apps CI/CD
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, synchronize, reopened, closed]
+    branches:
+      - main
+      
+jobs:
+  build_and_deploy_job:
+    if: github.event_name == 'push' || (github.event_name == 'pull_request' && github.event.action != 'closed')
+    runs-on: ubuntu-latest
+    name: Build and Deploy Job
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+          lfs: false
+      - name: Build And Deploy
+        id: builddeploy
+        uses: Azure/static-web-apps-deploy@v1
+        env:
+          CI: false
+          REACT_APP_LNBITS_NODE_URL: ${{ secrets.REACT_APP_LNBITS_NODE_URL }}
+          REACT_APP_LNBITS_USERNAME: ${{ secrets.REACT_APP_LNBITS_USERNAME }}
+          REACT_APP_LNBITS_PASSWORD: ${{ secrets.REACT_APP_LNBITS_PASSWORD }}
+          REACT_APP_LNBITS_ADMINKEY: ${{ secrets.REACT_APP_LNBITS_ADMINKEY }}
+          REACT_APP_LNBITS_STORE_ID: ${{ secrets.REACT_APP_LNBITS_STORE_ID }} 

```

Comment:
`REACT_APP_STORE_OWNER_EMAIL` is missing

NB Logically we should rename that to `REACT_APP_LNBITS_STORE_OWNER_EMAIL`.

Reply:
Added REACT_APP_STORE_OWNER_EMAIL and renamed to REACT_APP_LNBITS_STORE_OWNER_EMAIL

Reply:
Will also need to update the Rewards component to not break.

Reply:
already done ... it will come in my PR